### ISSUE TITLE
Auto fill 'override inform host'

### ISSUE THREAD

**Issue by rekcodocker** [linuxserver/docker-unifi-controller#159]
I love this Unifi container. It works very well. 

However, the first time it took me hours to get it to work.  
The point where you have to set 'Override inform host' just wasn't clear, I didn't even know what an 'inform host' was. 
So the APs were continously in an adopting-loop until I got this fixed.

Feature request: 
Could the container just fill this parameter at the initial installation?  
As I see it, it always needs to be filled, so it would be nice for new users if this could be automated.

**Comment by drizuid** [comment]
We wouldn't know what to put... it's per system. My inform host is different from yours. Our expectation is also that users read the documentation from the app devs, our readme mentioned the inform-host and the official documentation has a good amount of detail on what to do and how to do it.

tl;dr this isn't something we would automate as it would not work in 99% of cases
